src/System.Runtime.CompilerServices.Unsafe/ref/Configurations.props

```diff
@@ -0,0 +1,9 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <PropertyGroup>
+    <BuildConfigurations>
+      netstandard1.0;
+      netstandard;
```

Comment:
Which version out of these two will end up in the netcoreapp flat layout? 

It should be the netstandard1.0 one.